client/filter/tvs.density.js

```diff
@@ -128,6 +128,12 @@ function applyBrush(self, elem, brush) {
 
 	function getDecimalPlaces(value, min, max) {
 		let digits = 2
+		if (Math.abs(Math.trunc(value)) == 0) {
```

Comment:
Could this be simplified to `if (Math.abs(value) < 1)`?

Reply:
Actually there is no need for Math.abs here, I can remove it.

Reply:
Done